packages/turf-helpers/lib/geojson.ts

```diff
@@ -57,189 +8,3 @@ export type BBox = BBox2d | BBox3d;
  * the Feature object with the name "id", and the value of this member is either a JSON string or number.
  */
 export type Id = string | number;
-
-/**
- * Position
- *
- * https://tools.ietf.org/html/rfc7946#section-3.1.1
- * Array should contain between two and three elements.
- * The previous GeoJSON specification allowed more elements (e.g., which could be used to represent M values),
- * but the current specification only allows X, Y, and (optionally) Z to be defined.
- */
-export type Position = number[]; // [number, number] | [number, number, number];
-
-/**
- * Properties
- *
- * https://tools.ietf.org/html/rfc7946#section-3.2
- * A Feature object has a member with the name "properties".
- * The value of the properties member is an object (any JSON object or a JSON null value).
- */
-export type Properties = { [name: string]: any } | null;
-
-/**
- * Geometries
- */
-export type Geometries =
-  | Point
-  | LineString
-  | Polygon
-  | MultiPoint
-  | MultiLineString
-  | MultiPolygon;
```

Comment:
The downside of losing this, is that if you want to specify 'everything except GeometryCollection' (like a turf function that can handle all but GC's) there isn't a shorthand way to say that, you have to list all 6 other geometry types out.  In practice it doesn't come up much but handy.

Reply:
The upside of dropping this is that GeometryCollection is no longer excluded/treated differently, which under the geojson spec is a first class Geometry.  In the `geojson` package, the `Geometry` type includes all 7.